addons/binding/org.openhab.binding.chromecast/README.md

```diff
@@ -12,7 +12,12 @@ This can be configured on the binding level:
 
 ## Supported Things
 
-The binding currently supports the "classic" Chromecast that is an HDMI dongle as well as the Chromecast Audio, which only does audio streaming and offers a headphone jack.
+| Things             | Description                     | Thing Type |
```

Comment:
Please reformat the tables.

http://www.tablesgenerator.com/markdown_tables

Select 'File' > 'Past table data ...' and past the unformatted mark down table, you can then copy paste the formatted table 👍